lib/test_sites/listings.rb

```diff
@@ -5,10 +5,12 @@
 require 'json'
 
 module TestSites
+  # Wrapper class for an Evive locations.
```

Comment:
`Wrapper class for a test site location` would be more accurate. It can come from any source.